ui/src/components/policy/PolicyRow.js

```diff
@@ -510,6 +519,17 @@ export class PolicyRow extends React.Component {
                         verticalAlign={'text-bottom'}
                     />
                 </TdStyled>
+                {/*Tags*/}
+                <TdStyled align={center} width={'10%'}>
```

Comment:
ideally we have styling in the css file, is there a reason why styling is here?

Reply:
I dont know why its here but in all the policyRow page they styled to header this way

Reply:
ok for consistency then 👍